test/linter.js

```diff
@@ -82,13 +82,16 @@ async function getTagsFromConfig(config) {
 function execLinterCommand(args) {
   var cmd = `npx autorest@2.0.4152 --validation --azure-validator --message-format=json ${args}`.trim();
```

Comment:
we should just use autorest as a library and call the linter through it

Reply:
we use autorest as a library in the tooling service you can have a look there how to do it, that way also we wont have the autorest version hardcoded in the file

Reply:
Sure. I will look into it and take it up as a seperate issue.